index.js

```diff
@@ -232,10 +236,10 @@ async function HandleMessage(message, sender, respond) {
   cmd = cmd.toLowerCase();
  
   // removes the bang at the beginning
-  const command = cmd.slice(1)
-  if (Object.keys(dummyMapping).includes(command)) {
-    respond(dummyMapping[command](sender, args))
-  }
+  let command = cmd.slice(1)
+  const fallback = () => `!${command} is not a valid command`
+  const func = dummyMapping?.[command] ?? dummyMapping?.[aliases?.[command]] ?? fallback
```

Comment:
With this approach one could never rename the `!add` command to something else or disable the `!add` command.
I think it would be better to distinguish between `command names` and `commands` (and their aliases).
For example the `dummyMapping` could map a `command name` to a function, and then there is another map, that maps `commands` and aliases to command names:
For example there is the `add` command with the `addLevel` function, and then there is a `!add` and `!push` mapping for the `add` command.
With this approach the `add` command can also be configured to only be used with `!push`.
(`add` might be a bad example, but someone might want to disable the `!customcodes` command or the `!queue` command)

Reply:
agreed. I left the bangs out because I just don't like them tbh and it lets it shortcircuit non command messages faster

Reply:
`command names` and `commands` were just terms I came up with to explain better what I mean,
maybe calling it `function names` and `commands` would be better,
like `commands` could be mapped to `function names` and those are mapped to the actual functions,
whatever definition works
Yeah not having the bangs in there is probably better, but I left them in my explanation, because I thought I could bring across my point better